src/rgw/rgw_pubsub.cc

```diff
@@ -514,6 +597,15 @@ int RGWPubSub::Bucket::write_topics(const DoutPrefixProvider *dpp, const rgw_pub
 
 int RGWPubSub::get_topic(const DoutPrefixProvider *dpp, const std::string& name, rgw_pubsub_topic& result, optional_yield y) const
 {
+  if (driver->get_zone()->get_zonegroup().supports_feature(
+          rgw::zone_features::notification_v2)) {
+    const int ret = driver->read_topic(name, tenant, result, nullptr, y, dpp);
+    if (ret < 0) {
+      ldpp_dout(dpp, 1) << "ERROR: failed to read topic info for name: " << name
```

Comment:
```suggestion
      ldpp_dout(dpp, 1) << "failed to read topic info for name: " << name
```
depending with caller, this is not always an error (e.g. in topic creation)